src/index.ts

```diff
@@ -385,8 +389,14 @@ export async function cli(args: string[]) {
     namedExports: undefined,
     webDependencies: undefined,
   };
-  const doesWhitelistExist = !!webDependencies;
-  const arrayOfDeps = webDependencies || Object.keys(pkgManifest.dependencies || {});
+
+  let doesWhitelistExist = true;
+  const arrayOfDeps = [...webDependencies, ...scanImports(entry || '', {dest})];
```

Comment:
nit: if entry is undefined, it would be nice to just not call scanImports at all, so that we don't have to rely on it's "if undefined" behavior doing the right thing. What about something like this?

```
const arrayOfDeps = [...(webDependencies || [])];
if (entry) {
  arrayOfDeps.push(...scanImports(entry, {dest}));
}
if (!arrayOfDeps.length) {
  // ...
}
```
```